mmv1/products/dns/api.yaml

```diff
@@ -276,6 +276,13 @@ objects:
                 description: |
                   The fully qualified URL of the service directory namespace that should be
                   associated with the zone. Ignored for `public` visibility zones.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'cloudLoggingConfig'
+        description: 'Cloud logging configuration'
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'enableLogging'
```

Comment:
typically when there is only one field in a nested object, we'll make the field `required: true`. Especially in the case where `enableLogging` will always need to be specified one way or another.

Reply:
That makes sense. Changed.